package.json

```diff
@@ -99,6 +99,7 @@
 		"@wordpress/icons": "^10.0.0",
 		"array-move": "^4.0.0",
 		"clsx": "^2.1.1",
+		"dompurify": "^3.2.6",
```

Comment:
Lets use https://github.com/WordPress/gutenberg/tree/de11ef92c9f6478fedb7bd5f56b1612943d5c7a9/packages/dom#safehtml instead to avoid the additional dependency